package-lock.json

```diff
@@ -1,4049 +1,8 @@
 {
 	"name": "github-graphql-nb",
-	"version": "0.0.8",
-	"lockfileVersion": 2,
+	"version": "0.1.0",
+	"lockfileVersion": 1,
```

Comment:
Ah I forgot to bump the extension version in the lockfile. Still, the lockfile probably doesn't need to change this much (VS Code develops against node 16.x, which comes with npm 7, and which generates v2 lockfiles). Would you mind either updating npm and rerunning `npm i`, or dropping the change to the package-lock.json? Thanks a ton!

Reply:
If I update `cross-fetch` and `node-fetch` I'm still seeing vuln warnings on mocha, but being a breaking change.  Leaving that one alone.

Reply:
Fixed this as best as I understand it.